tests/subsys/bootloader/bl_crypto/testcase.yaml

```diff
@@ -1,24 +1,38 @@
 tests:
   bootloader.bl_crypto:
     sysbuild: true
-    platform_allow: nrf52840dk/nrf52840 nrf52dk/nrf52832 nrf9160dk/nrf9160
-      nrf5340dk/nrf5340/cpuapp nrf52833dk/nrf52833
-    integration_platforms:
+    platform_allow:
+      - nrf52833dk/nrf52833
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
-      - nrf9160dk/nrf9160
       - nrf5340dk/nrf5340/cpuapp
+      - nrf9151dk/nrf9151
+      - nrf9160dk/nrf9160
+      - nrf9161dk/nrf9161
+    integration_platforms:
       - nrf52833dk/nrf52833
+      - nrf52840dk/nrf52840
+      - nrf52dk/nrf52832
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf9151dk/nrf9151
+      - nrf9160dk/nrf9160
+      - nrf9161dk/nrf9161
     tags: b0 sysbuild
   # Deprecated child and parent build to ensure feature does not break until it is removed
   bootloader.bl_crypto.child_parent:
     sysbuild: false
-    platform_allow: nrf52840dk/nrf52840 nrf52dk/nrf52832 nrf9160dk/nrf9160
-      nrf5340dk/nrf5340/cpuapp nrf52833dk/nrf52833
-    integration_platforms:
+    platform_allow:
+      - nrf52833dk/nrf52833
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
-      - nrf9160dk/nrf9160
       - nrf5340dk/nrf5340/cpuapp
+      - nrf9160dk/nrf9160
```

Comment:
Missing 9151? Although, given that this is deprecated, maybe there is no need for 9161 either.

Reply:
Added 9151 since the test was deprecated recently.